estate/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Real Estate',
+    'version': '1.0',
```

Comment:
```suggestion
    'version': '16.0.0.0.0',
```

Reply:
According to the documentation of OCA on [version numbers](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers) "`12.0.1.0.0` is expected for the first release of an `12.0 module`", so this module should be `16.0.1.0.0` and move forward following the "semantics `breaking.feature.fix`". In [this commit](https://github.com/OCA/crm/blame/f462ddef4993f79917dd54b51177f9177389d74f/crm_location/__manifest__.py) you can see that the first ever released version of the module `crm_location` was version `10.0.1.0.0`.